Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM python:3.8
-LABEL maintainer="Robin Cole @robmarkcole"
+FROM malenurse/ubuntu-20.04-base
```

Comment:
I would rather this was a stock base image with the required build instructions in a Dockerfile,arm64 that was in this repo, since then maintenance is contained within this repo

Reply:
Oof. sorry about that, I was just using this as a shortcut for testing. This is only my third PR, and first that I actually had to go back and tinker; I didn't realize that the repos were linked concurrently through this. *Really* sorry if your email's been blowing up because of it. My Pi is building the rest of Streamlit now, and then I'm going to look at how much I can slim down the installed packages that were needed build pyarrow since they really bloat the image. I'll keep everything offline until I'm actually ready to send it over a little later, but I should have a solution in a few hours that doesn't require much more than adding the instructions for building pyarrow from source to the current Dockerfile. Hopefully won't even  have to rebase since python:3.8 is running a very similar distro to Ubuntu. Again, *really* sorry for those annoyances.

Reply:
No problem :-)